client/js/components/map/admin/LayerSettings.vue

```diff
@@ -351,6 +356,64 @@ export default {
       })
     },
 
+    validateXtrasseUrl () {
+      // UX feedback via notifications only - backend does authoritative validation
+      if (!this.url || this.url === '') {
+        return true
+      }
+
+      const collectionsPattern = '/collections/'
+      const lowerUrl = this.url.toLowerCase()
+      const collectionsIndex = lowerUrl.indexOf(collectionsPattern)
+
+      // Check if URL contains /collections/ (case-insensitive)
+      if (collectionsIndex === -1) {
+        const errorMessage = Translator.trans('error.map.layer.xtrasse.missing.collections')
+        dplan.notify.error(errorMessage)
+        return false
+      }
+
+      // Check if /collections/ is not at the end (there must be content after it)
+      const afterCollections = this.url.substring(collectionsIndex + collectionsPattern.length)
+      if (afterCollections.trim() === '' || afterCollections === '/' || afterCollections.match(/^\/+$/)) {
```

Comment:
nitpick: I think it normally improves readability to store more complex conditions in a variable with a descriptive name